### DISCUSSION TITLE
svgSalamander dependency?

### DISCUSSION THREAD

**Discussion by MSchmidke** [mapsforge/mapsforge#1216] [Q&A] [answered]
Hi all,

I'm trying to add mapsforge-map-awt as a maven dependency, but this doesn't work because svgSalamander cannot be resolved.

It seems that svgSalamander has moved or something - I can see commit 935fb99c which changes svgSalamander dependency. Seems that will be contained in mapsforge release 0.16.0; latest available on maven is 0.15.0.

Are there any plans when it will be released? Or any other workaround I can do?

Regards 

Marcus.

**Comment by devemux86** [comment, marked as answer]
I updated SVG Salamander dependency for next Mapsforge version.

Depending on your Mapsforge version you can follow the appropriate instructions:
- [Mapsforge 0.15.0 integration guide](https://github.com/mapsforge/mapsforge/blob/0.15.0/docs/Integration.md#desktop) (you need to add JitPack repository)
- [Mapsforge snapshot integration guide](https://github.com/mapsforge/mapsforge/blob/master/docs/Integration.md#desktop)

**Reply by MSchmidke** [reply]
Great! Works! Thank you for quick reply!! Adding JitPack repository to pom was the solution. Sorry, I am not very skilled using maven.
